### PR TITLE
Table is now responsive, mobile-ready, new mobile navigation with ani…

### PR DESCRIPTION
**Pull request by mcintosh101**
- Added Mobile menu
- Fixed responsive issues, should be fine on most devices now
- Made the table responsive, it's now vertical on smaller screens
-- Table still has dummy rows so you can test your end, these need to be removed.
-- `<td>` elements now requires a `data-label` attribute to work correctly on smaller screens
- New scripts.js to control the mobile menu animations
- Font-sizes based on a single base value
- z-index controlled through CSS vars
- Fixed navigation sub-menu positioning issues on bigger screens

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "cSpell.words": [
+        "initialised",
+        "intialise",
+        "punchcard"
+    ]
+}
```
Comment (w-martin) [inline .vscode/settings.json]: Please remove this / add to .gitignore

File: templates/styles.css [hunk 2]
```diff
@@ -37,26 +44,23 @@ body {
...
  **/
 h1 {
-    font-size: 48px;
+    font-size: calc(var(--base-font-size) * 6);
     font-weight: 600;
 }
 
 h4 {
-    font-size: 24px;
+    font: calc(var(--base-font-size) * 3);
```
Comment (w-martin) [inline templates/styles.css]: Should this be `font-size` ?

File: templates/styles.css [hunk 3]
```diff
@@ -147,31 +203,51 @@ nav ul ul {
...
+    right: -10px;
+    
+    content: '\f0d7';
+    
+    color: var(--primary-colour);
+    font: normal normal normal 14px/1 FontAwesome;
+    font-size: inherit;
+    text-rendering: auto;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```
Comment (w-martin) [inline templates/styles.css:241]: Does that only apply on `osx`?

File: templates/tracking.html [hunk 4]
```diff
@@ -1,128 +1,221 @@
...
+                            <td data-label="Objectives">
+                                Test
+                            </td>
+                            <td data-label="Authors">
+                                Test
+                            </td>
+                        </tr>
+                        <!-- END: Dummy data to show how the table should be built -->
+                        
+                        <!-- {monitors} -->
```
Comment (w-martin) [inline templates/tracking.html]: Aw. Fancy modifying `reporting.py` to output those extra labels?

**Review by w-martin — CHANGES_REQUESTED** [review]
Awesome, thanks for the PR! A couple of requested changes, and a potential typo. Need to work out how to fork your PR if you don't fancy modifying `reporting.py`

File: .vscode/settings.json (hunk 1 above)
Comment (mcintosh101) [reply .vscode/settings.json]: Will do 👍

File: templates/styles.css (hunk 3 above)
Comment (mcintosh101) [reply templates/styles.css:241]: Yeah, they are common vendor specific properties used to make the text look sharper across different browsers, some more info can be found here if you're interested: https://davidwalsh.name/font-smoothing

File: templates/tracking.html (hunk 4 above)
Comment (mcintosh101) [reply templates/tracking.html]: All done.

**Comment by mcintosh101** [comment]
All requested changes have been pushed. 👍

**Review by mcintosh101 — COMMENTED** [review]
All done

File: templates/tracking.html (hunk 4 above)
Comment (w-martin) [reply templates/tracking.html]: Thanks, now could the the dummy data be removed / commented out so that `{monitors}` is used?

**Review by w-martin — CHANGES_REQUESTED** [review]
All good :+1: just need to comment out the dummy data in `summary.html` and put back in `{monitors}`, and fix the `font` -> `font-size`

**Review by mcintosh101 — COMMENTED** [review]
All should be done now, removed dummy code and updated font property.

**Review by w-martin — APPROVED** [review]
:+1:

**Pull request merged**